### ISSUE TITLE
[FEATURE] When running pipelines or annotating full frame annotations use filled bars in timeline

### ISSUE THREAD

**Issue by mattdawkins** [VIAME/VIAME-Web#539]
e.g. in the attatched all of those empty areas would be filled with blue (instead of just blue border)

![Screenshot at 2021-01-21 16-22-42](https://user-images.githubusercontent.com/2353972/105427790-ecb3a280-5c1b-11eb-8b44-cfcab78b4e67.png)

**Comment by subdavis** [comment]
The "Events" view is the correct timeline view for this.   I believe this feature request tries to make the histogram do event things.

The histogram view is nearly worthless for this dataset.

**Comment by subdavis** [comment]
This would involve changing the meaning of the histogram, making it into a stacked bar chart at each frame rather than the current thing, which is a bunch of bar charts superimposed on each other.

**Comment by waxlamp** [comment]
@mattdawkins, I don't think this issue is actionable as written. Can you provide more motivation for why you need this? I'd like to consider reworking this into the events view as Brandon is suggesting, or else gathering more input from you on what problem you're trying to solve.
